analyticsdata/src/run_report_with_pagination.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Copyright 2022 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Google Analytics Data API sample application demonstrating the use of
+ * pagination to retrieve large result sets.
+ * See https://developers.google.com/analytics/devguides/reporting/data/v1/rest/v1beta/properties/runReport#body.request_body.FIELDS.offset
+ * for more information.
+ * Usage:
+ *   composer update
+ *   php run_report_with_pagination.php YOUR-GA4-PROPERTY-ID
+ */
+
+namespace Google\Cloud\Samples\Analytics\Data;
+
+// [START analyticsdata_run_report_with_date_ranges]
+use Google\Analytics\Data\V1beta\BetaAnalyticsDataClient;
+use Google\Analytics\Data\V1beta\DateRange;
+use Google\Analytics\Data\V1beta\Dimension;
+use Google\Analytics\Data\V1beta\Metric;
+use Google\Analytics\Data\V1beta\MetricType;
+use Google\Analytics\Data\V1beta\RunReportResponse;
+
+/**
+ * @param string $propertyID Your GA-4 Property ID
+ * Runs a report several times, each time retrieving a portion of result
+ * using pagination.
```

Comment:
PHPDoc tags should be last
```suggestion
 * Runs a report several times, each time retrieving a portion of result
 * using pagination.
 * @param string $propertyId Your GA-4 Property ID
```